src/templates/widget_index_case.html

```diff
@@ -16,19 +16,19 @@
     <div class="form-group">
       <div class="col-sm-11">
         <input class="fd-index-property-source form-control"
-               value="<%-v['calculate']%>" type="text" placeholder="Calculation" />
+               value="<%-v['calculate']%>" type="text" placeholder="Referenced Case ID" />
```

Comment:
Yeah. it uses a bind instead of saving in place. It's technically the same as a "calculate condition" from a hidden value, but I don't see a time when you wouldn't want it to be a "referenced case id"